content/blog/2021-03-15-March-21-dvc-heartbeat.md

```diff
@@ -177,13 +177,19 @@ image="/uploads/images/2021-03-15/arxiv.png"/>
 
 ## Tweet Love ❤️
 
-From a Porutguese speaking community member in Finland...
+From a Portuguese speaking community member in Finland...
```

Comment:
this jumped out at me and I figured I should address it while we're already changing the past here.